superchain/superchain.go

```diff
@@ -60,8 +60,8 @@ type HardForkConfiguration struct {
 type SuperchainLevel uint
 
 const (
-	Standard SuperchainLevel = 2
-	Frontier SuperchainLevel = 1
+	Standard SuperchainLevel = 1
+	Frontier SuperchainLevel = 0
```

Comment:
We did well here to define these constants, this diff is much smaller than it would need to be if we had sprinkled `0`s and `1`s everywhere!